workflows/short-read-mngs/test/test_short_read_mngs.py

```diff
@@ -31,7 +31,7 @@ def test_bench3_viral(short_read_mngs_bench3_viral_outputs):
         taxon_counts = json.load(infile)["pipeline_output"]["taxon_counts_attributes"]
 
     taxa = set(entry["tax_id"] for entry in taxon_counts)
-    assert len(taxa) == 177
+    assert abs(len(taxa) - 184) < 16
```

Comment:
Are we expecting to introduce some variability into the host filtering? Or was this just a holdover from the old assert?